src/panels/config/backup/dialogs/dialog-upload-backup.ts

```diff
@@ -154,7 +162,7 @@ export class DialogUploadBackup
 
     this._uploading = true;
     try {
-      await uploadBackup(this.hass!, file, agentIds);
+      await uploadBackup(file, agentIds, this.hass);
```

Comment:
`hass` is optional right? Should be typed correctly.

Reply:
No why? backup panel will always have hass right?

Reply:
This is also used during onboarding right? Then there is no hass? If it is not optional, why did you move the hass param to the end?

Reply:
Ah ok, no it's not used. I did used it but @piitaya suggested to create a seperate function. I'll revert it :+1: